### PR TITLE
edge to master

### PR DESCRIPTION
**Pull request by budgetpreneur**

### CODE REVIEW

File: src/UnitTests/PubnubApi.Tests/WhenAMessageIsPublished.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public static void Init()
 
             if (!PubnubCommon.PAMEnabled) { return; }
 
-            if (PubnubCommon.PAMEnabled && !string.IsNullOrEmpty(PubnubCommon.SecretKey))
+            if (PubnubCommon.PAMEnabled && string.IsNullOrEmpty(PubnubCommon.SecretKey))
```
Comment (davidnub) [inline src/UnitTests/PubnubApi.Tests/WhenAMessageIsPublished.cs]: Out of curiosity was the removal of the `!` intentional here?

Comment (budgetpreneur) [reply src/UnitTests/PubnubApi.Tests/WhenAMessageIsPublished.cs]: yes @davidnub  that was intentional change. When Access Manager is enabled and Secret Key is NOT provided, it does not make sense to process in calling Grant() method, else non-200 status code will be returned.

Comment (davidnub) [reply src/UnitTests/PubnubApi.Tests/WhenAMessageIsPublished.cs]: Ah ok, thanks for the explanation.

Comment (budgetpreneur) [reply src/UnitTests/PubnubApi.Tests/WhenAMessageIsPublished.cs]: cool. If satisfied with the PR, please approve my PR.

**Pull request merged**